### DISCUSSION TITLE
How to apply cardStyleInterpolator to stack from createNativeStackNavigator

### DISCUSSION THREAD

**Discussion by mohsinulhaq** [software-mansion/react-native-screens#808] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/7688231/107284862-60f79e00-6a84-11eb-929d-188f9b90cbfc.png)
How can I apply `cardStyleInterpolator` to stacks created via `createNativeStackNavigator`? I see that it's an option available to stacks created via `createStackNavigator`. Is it possible to something similar with the native stack?

**Comment by WoLewicki** [comment, marked as answer]
There is no option for this for now other than using transitions available in `stackAnimation`. You can check https://github.com/software-mansion/react-native-screens/issues/317 as there is a discussion about things that are kind of similar to what you ask for.
